book/09-git-and-other-scms/sections/client-svn.asc

```diff
@@ -330,14 +330,14 @@ No changes between 71af502c214ba13123992338569f4669877f55fd and refs/remotes/ori
 Resetting to the latest refs/remotes/origin/trunk
 ----
 
-Running `dcommit` on a branch with merged history works fine, except that when you look at your Git project history, it hasn't rewritten either of the commits you made on the `experiment` branch – instead, all those changes appear in the SVN version of the single merge commit.
+Ang pagpapatakbo ng `dcommit` sa isang branch na may na-merge na kasaysayan ay gagana ng maayos, maliban kapag ikaw ay tumingin sa iyong kasaysayan ng Git na proyekto, hindi nito sinulatang muli ang alinman sa mga commit na ginawa mo sa `experiment` na branch – sa halip, ang lahat ng mga pagbabagong iyon ay magpapakita sa SVN na bersyon ng solong merge na commit.
 
-When someone else clones that work, all they see is the merge commit with all the work squashed into it, as though you ran `git merge --squash`; they don't see the commit data about where it came from or when it was committed.
+Kapag may ibang tao na magko-clone ng trabahong iyon, lahat ng makikita nila ay merge na commit kasama ang lahat na trabahong napipi sa iyon, kagaya ng pagpatakbo mo ng `git merge --squash`; hindi nila makikita ang commit na datos tungkol sa kung saan ito nanggaling o kailan ito na-commit.
```

Comment:
"makikita nila ay merge na commit" to "makikita nila ay ang merge na commit"... simply add "ang".

I'm not sure with "napipi" translation for "squashed"??? what's your take on this @severinolorillajr ?

Reply:
- changing it now
- I too am a bit confused with the translation with this one, I think in this context "squashed" also means "crammed". And one of the closest translation for "squashed" is "napipi". Or in this context we could use "naisiksik" perhaps?

Reply:
Changing it to naisiksik now.